mobile/src/main/java/org/openhab/habdroid/ui/FullscreenImageActivity.kt

```diff
@@ -127,7 +144,21 @@ class FullscreenImageActivity : AbstractBaseActivity() {
         bitmap ?: return finish()
 
         Handler(Looper.getMainLooper()).post {
```

Comment:
Now that I see it, why does this use a Handler instead of `launch(Dispatchers.Main)`?

Reply:
`loadImage()` does some IO, so I originally thought it's better to use the `Dispatchers.IO` here.